### DISCUSSION TITLE
Feature: Checkbox / CheckboxGroup or RadioGroup prop "multiple"

### DISCUSSION THREAD

**Discussion by mockingjet** [tailwindlabs/headlessui#517] [Ideas]
Adding a "multiple" prop to RadioGroup could be a quick solution. It would be better to have two separate components.

**Comment by ahoiroman** [comment]
Second this. Can't wait to get rid of my ugly workaround ;-)

**Comment by akorajac** [comment]
I guess this wont happen ?

**Comment by matt-cassinelli** [comment]
This would be pretty neat
